sections/discussion/future-work.tex

```diff
@@ -11,6 +14,21 @@ \section{Future work}
     \item Extend the recommender to have content based and collaborative approach.
 \end{itemize}
 \noindent
+\textit{Update the select multiple button}:
+During the \autoref{sec:usability-test-section} we found that our current select multiple were unintuitive. 
+They didn't seem to understand that you were able to select multiple languages, as it only highlighted the selected items with a grey color.
+These select multiple buttons should be edited, so that it becomes more intuitive.
```

Comment:
```suggestion
These select multiple buttons should be edited, so that it becomes more intuitive, or a more clear description should be written prior to selecting, properly indicating that it is possible to select multiple options.
```